Bottom: a86660274c72f2eb72ab88afaa1bcc3179db4380
Top:    aeb24633aaa003c8c913afb8e47f284008fa114e
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 23:43:20 -0500

Refresh of readme

---

diff --git a/README.md b/README.md
index 0b56ff6..6238caa 100644
--- a/README.md
+++ b/README.md
@@ -6,6 +6,7 @@ the original documentation page at: https://www.dokuwiki.org/plugin:tablecalc.
 
 Forked from https://github.com/pirati-cz/tablecalc. Adds the following
 functions:
+
   | ''tag(name)''     | Associate the current row and column with a name |
   | ''tag_col(name)'' | Return the column for the specified tag          |
   | ''tag_row(name)'' | Return the row for the specified tag             |
